django_project/localities_osm/utilities.py

```diff
@@ -1,8 +1,86 @@
 __author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
 __date__ = '04/04/19'
 
-from localities_osm.models.locality import LocalityOSMView
+from localities_osm.models.locality import LocalityOSMView, LocalityOSM
 
 
 def get_all_osm_query():
     return LocalityOSMView.objects.filter(osm_id__isnull=False).exclude(name='')
+
+
+def convert_into_osm_dict(locality):
+    """Utility to convert old locality dict into osm dict.
+
+    :param locality: Locality object.
+    :type locality: localities.models.Locality
+
+    """
+
+    data = locality.repr_dict()
+
+    osm_dict = data['values']
+    osm_dict['name'] = data['name']
+
+    if osm_dict.get('staff', None):
+        staff = osm_dict['staff'].split('|')
+        staff = [u'0' if v is u'' else v for v in staff]
+        osm_dict['staff_doctors'] = staff[0]
+        osm_dict['staff_nurses'] = staff[1]
+        del osm_dict['staff']
+
+    if osm_dict.get('inpatient_service', None):
+        beds = osm_dict['inpatient_service'].split('|')
```

Comment:
hi @ann26 
i'm sorry, but beds[0] should be into osm_dict['beds']
and the second one, beds[1], should be osm_dict['partial_beds'] (which is extension)
please update this